go-fuzz-build/main.go

```diff
@@ -90,6 +108,12 @@ func main() {
 	if *flagLibFuzzer && *flagRace {
 		c.failf("-race and -libfuzzer are incompatible")
 	}
+	if checkModVendor() {
+		// Preliminary modules support: we don't support -mod=vendor.
+		// Part of the issue is go-fuzz-dep and go-fuzz-defs
```

Comment:
Would fixing #252 help? Just curious.

Reply:
My guess is #252 would mean this error check is no longer needed. Addressing #252 should at least cause the current failure that happens with vendoring to be resolve. (Of course, until we see it happen, my guess could be wrong, or there could be another failure hiding behind that first failure, but my guess is #252 would be sufficient).

That is why I was looking into #252, and that is also why I posted in https://github.com/dvyukov/go-fuzz/issues/195#issuecomment-539161466 a few weeks ago that #252 would be helpful for anyone else to address if they were waiting on modules support, or waiting on this PR. (I might get to #252 at some point, but not likely soon).

Reply:
I looked briefly again at #252 today as part of reviewing this. I don't think the existing go/packages support is sufficient. At least, I could see how to do it. I couldn't get go/packages to recognize new packages added via an overlay. And as a conceptual matter, I don't see how go/packages could know what the package path of an overlay-only package is.